interfaces/cython/cantera/examples/reactors/NonIdealShockTube.py

```diff
@@ -1,7 +1,5 @@
 # coding: utf-8
 """
-Non-Ideal Shock Tube Example
```

Comment:
Isn’t that detail (non-ideal) relevant?

Reply:
I would assume we aren't going to arbitrarily rename the file to remove that information, though. The point of removing this line is so that the following paragraph, "Ignition delay time computations in a high-pressure reflected shock tube reactor" will be used as the extended description on [the examples index](https://cantera.org/examples/python/index.html#python-example-reactors), rather than just uselessly repeating the file name.

Reply:
I'm not opposed to eliminating the redundant paragraph, but the information could be easily integrated in the subsequent paragraph as:

> Ignition delay time computations in a high-pressure reflected shock tube
reactor *assuming non-ideal gas behavior*.